services/director/src/simcore_service_director/api/v0/schemas/project-v0.0.1.json

```diff
@@ -252,6 +252,21 @@
                 }
               }
             },
+            "inputsUnits": {
+              "type": "object",
+              "description": "values of input unit",
+              "patternProperties": {
+                "^[-_a-zA-Z0-9]+$": {
```

Comment:
TIP: A simpler way to write that regex is ``^[-\w]+$``

Unfortunately this regex for the input key allows things like ``my-input`` or ``0-input`` which cannot be used as variable names ... which is inconvenient. Since these key names are not accessible to the user I think there is no reason to keep this regex and replace it instead by ``models_library.basic_regex.PUBLIC_VARIABLE_NAME_RE`` .  So in the json-schema we should start adding ``"^[^_\W0-9]\w*$"``